spec/classes/init_spec.rb

```diff
@@ -725,12 +593,7 @@
 
       case facts[:os]['family']
       when 'RedHat'
-        if facts[:os]['release']['major'].to_i == 6
-          context 'On a redhat 6 based OS' do
-            it { is_expected.to contain_class('consul').with_init_style('redhat') }
-            it { is_expected.to contain_file('/etc/init.d/consul').with_content(%r{daemon --user=consul}) }
-          end
-        elsif facts[:os]['release']['full'] == '2016.09'
+        if facts[:os]['release']['full'] == '2016.09'
           context 'On an Amazon based OS' do
             it { is_expected.to contain_class('consul').with_init_style('redhat') }
             it { is_expected.to contain_file('/etc/init.d/consul').with_content(%r{daemon --user=consul}) }
```

Comment:
I'd expect these to also fail now?

Reply:
But I guess they are not even executed, as that is not listed as `supported os`?

Reply:
Ah, because we don't support that OS and it's not listed in the metadata.json, so the OS is never set to 2016.19. I cleaned it up.